.github/renovate.json

```diff
@@ -14,12 +14,6 @@
       "stabilityDays": 3,
       "reviewers": ["team:sig-ux"]
     },
-    {
```

Comment:
Bootstrap has been moved into its own SCSS file (just for the setup wizard), where it can eventually be removed.